components/src/core/components/TableFilter/__tests__/__snapshots__/table-filter.spec.ts.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TableFilter Renders 1`] = `
+<div class="oxd-table-filter">
+  <div class="oxd-table-filter-header spilled">
```

Comment:
**Spilled** class should not be there by default. This is an outcome of not finding the element inside the doc. It becomes 16 by default.This should be corrected